tests/providers/ssh/hooks/test_ssh.py

```diff
@@ -1028,6 +1010,24 @@ def test_command_timeout_fail(self):
                     None,
                 )
 
+    def test_command_timeout_not_set(self):
```

Comment:
this illustrates how to set _infinite_ timeout

Reply:
Looks like with your changes when `cmd_timeout` is not set in hook/operator, but set to `null` in connection extra it still means "infinite timeout", so for me it's ok.

Reply:
correct, you can set it to `null` in connection extra or `None` when you create the hook. Also i think after we replace the default value from `None` to `ArgNotSet` in `SSHOperator` as suggested by @uranusjr, we should also be able to set it as the operator level.